webpack.config.js

```diff
@@ -47,5 +47,6 @@ module.exports = {
   devServer: {
     contentBase: './dist',
     historyApiFallback: true,
+    disableHostCheck: true, // That solved it
```

Comment:
I don't get this comment, can you specify what was fixed?

Reply:
I forgot to remove the comment. This bypass flag seems necessary for webpack-dev-server to work properly.